app/views/smart_proxies/logs/_modules_card.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="card-pf card-pf-accented card-pf-aggregate-status card-pf-with-action">
+  <h2 class="card-pf-title">
+    <a href="#" data-toggle="tooltip" data-placement="top" title="<%= features.join(', ') %>"><%= n_('%s Features active', '%s Features active', features_started) % features_started %></a>
```

Comment:
The singular string passed to `n_()` is plural here, shouldn't it be "Feature active" (and probably lower case, since it's the second word)?